Big Integers/Big Sorting.cpp

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Please follow the same code format as the other programs. Use [this program](https://github.com/AjitZero/algo-cpp/blob/master/Dynamic%20Programming/LongestCommonSubsequence.cpp) as reference.

Reply:
```
/**
 * 
 * Title: <Specific title>
 * Description: <Specific description>
 * Author: <Your name>
 * GitHub: <Your link>
 * Twitter: <Your link (optional)>
 * 
 */
```